src/libmongoc/src/mongoc/mongoc-crypt.c

```diff
@@ -1428,6 +1429,13 @@ _mongoc_crypt_new (const bson_t *kms_providers,
       goto fail;
    }
 
+   if (cache_expiration_ms.set) {
+      mongocrypt_setopt_key_expiration (crypt->handle, cache_expiration_ms.value);
```

Comment:
Suggest doing `git merge master` to update `compile-libmongocrypt.sh` to [use libmongocrypt 1.12.0](https://github.com/mongodb/mongo-c-driver/blob/70d204fd3b46d710894e2f94127567c0a69dae86/.evergreen/scripts/compile-libmongocrypt.sh#L13). libmongocrypt 1.12.0 contains the needed `mongocrypt_setopt_key_expiration` symbol.